src/utils/common.js

```diff
@@ -49,6 +49,14 @@ export default {
         let index = Converters.byteArrayToSignedInt32(indexBytes)
         return index
     },
+    getContractKeyString(index, key) {
+        let stateIndex = new Uint8Array([index, 2])
+        let addrBytes = Base58.decode(key)
```

Comment:
We cannot simply decode base58 string here. We must covert the bytes according to DataEntry type of key. For example, to query channel creator of a payment channel by channel ID, we need input the key as `SHORT_BYTES_TYPE`.